internal/pkg/term/selector/selector.go

```diff
@@ -79,15 +79,16 @@ type AppEnvLister interface {
 	ListApplications() ([]*config.Application, error)
 }
 
-// ConfigSvcLister wraps the method to list svcs in config store.
-type ConfigSvcLister interface {
+// ConfigWlLister wraps the method to list workloads in config store.
+type ConfigWlLister interface {
```

Comment:
Nit: can we rename to ConfigWorkloadLister. I find Wl difficult to parse, I think it's fine if we have wl for private variables but we should consider spelling it out for exported entities

Reply:
yup, done!